unabridged-works-volume-4/original/03-queries-and-animadversions.adoc

```diff
@@ -911,16 +912,18 @@ I am a lover of souls, and an earnest desirer after their welfare;
 but especially of such as breathe after and long for communion with the Lord,
 in that which is living and pure.
 
-ISAAC PENINGTON
+[.signed-section-signature]
+Isaac Penington
 
+[.signed-section-context-close]
 25th of the 11th month, 1667
 
-=== A REPLY TO THY ANIMADVERSIONS
+=== a Reply to Thy Animadversions
```

Comment:
the `a` here should be capitalized.

Also, while you’re there, can you add a `[.centered]` to this heading as well? Most of the time when I've had a heading immediately followed by a `.inline` i've been centering the first one. I must have missed this one.